### PR TITLE
fix(10_linux): gather package facts and check for postfix existence

### PR DESCRIPTION
**Pull request by s3lph**

### CODE REVIEW

File: roles/maintenance_10_linux/tasks/main.yml [hunk 1]
```diff
@@ -165,13 +168,14 @@
     regexp: "^root:"
     line: "root: {{ linux_serverlogs_root_alias }}"
     insertbefore: EOF
+  when: "'postfix' in ansible_facts.packages"
```
Comment (Daenou) [inline roles/maintenance_10_linux/tasks/main.yml:171]: Is /etc/aliases _only_ used by postfix in every distribution?

Comment (s3lph) [reply roles/maintenance_10_linux/tasks/main.yml:171]: No, it's used by sendmail and exim as well. However, our serverlogs setup uses postfix, and I'd argue that it thus doesn't make much sense to check for a serverlogs alias if postfix is absent.

**Comment by Daenou** [comment]
Please also change the appropriate lines in the README file.

**Review by Daenou — APPROVED** [review]
LGTM :rocket:

**Pull request merged**